src/02-crm/addresses.apib

```diff
@@ -0,0 +1,21 @@
+## Addressess [/levelTwoAreas]
+
+We provide a set of data which can be used to build addresses
+
+### levelTwoAreas.list [GET /levelTwoAreas.list]
+
+Gets all level two areas (which correspond to provinces, departments or states in most countries).
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + country: `BE` (string, required)
+        + language: `nl` (string, optional) - if not passed, we return the name in the primary language of the country
```

Comment:
Q: Should we make it passive like the other descriptions ?
 `if not passed,  the name is returned in the primary language of the country`